src/components/organisms/SearchBar.vue

```diff
@@ -61,18 +50,22 @@ export default defineComponent({
         geoJsonShape.value = JSON.stringify(specifiedShape.value);
       }
     }, { deep: true });
-    watch(searchResults, (newList, oldList) => {
-      if (oldList === undefined && newList && newList.length > 0) {
-        toggle();
+    watch(searchResults, (newValue, oldValue) => {
+      if (oldValue?.length !== newValue?.length) {
```

Comment:
Is the length the right thing to check here? Or do we actually want to trigger this whenever the search results have changed at all? I'm thinking of the case where a new search was performed and has different results, but coincidentally has the same number of results as the previous search.

Reply:
Why have this at all? If we are `watch`ing `searchResults`, then this happens anytime the results are updated and thus we probably want to switch to this tab?

Reply:
Suggestion:
```ts
    watch(searchResults, (newValue, oldValue) => {
      reveal.value = true;
      selectedTab.value = 'results';
    });
```